modules/nw-about-multicast.adoc

```diff
@@ -14,14 +14,23 @@ discovery and not a high-bandwidth solution.
 ====
 
 Multicast traffic between {product-title} Pods is disabled by default. If you
-are using the OpenShiftSDN network plug-in, you can enable multicast on a
+are using the OpenShift SDN network plug-in, you can enable multicast on a
 per-project basis.
 
-When using the OpenShiftSDN network plug-in:
+When using the OpenShift SDN network plug-in in `networkpolicy` isolation mode:
 
 * Multicast packets sent by a Pod will be delivered to all other Pods in the
-project, regardless of `NetworkPolicy` objects. Pods may be able to communicate
+project, regardless of NetworkPolicy objects. Pods may be able to communicate
 over multicast even when they cannot communicate over unicast.
 * Multicast packets sent by a Pod in one project will never be delivered to Pods
-in any other project, even if there are `NetworkPolicy` objects allowing
+in any other project, even if there are NetworkPolicy objects allowing
 communication between the projects.
+
+When using the OpenShift SDN network plug-in in `multitenant` isolation mode,
+multicast packets sent by a Pod in one project will be delivered to Pods in
+another project only if the following conditions are true:
+
+* Each project must be joined together to allow multicast packets to traverse
+between Pods.
+* Each project must have multicast enabled.
```

Comment:
This is true, but sort of backwards. The "joined together" should be part of the "question", not part of the "answer": When using the OpenShift SDN network plug-in in `multitenant` isolation mode, if you join two (or more) projects together with `oc admin pod-network join-projects`, then if you want to enable multicast, you must enable it in all of the joined projects.

Reply:
@danwinship, it may make more sense in context:

Preview build: https://multicast-with-multitenant--ocpdocs.netlify.com/openshift-enterprise/latest/networking/openshift-sdn/using-multicast.html#nw-about-multicast_using-multicast

But maybe not?

Reply:
That is better but it still implies that you can enable multicast in a subset of the projects, which is not true. If there are `n` projects joined together, and you enable multicast in anywhere from `0` to `n-1` of them, it will have no effect; multicast will not be enabled. Only when you enable multicast in all `n` namespaces does it take effect, and multicast is enabled across the entire set of joined namespaces.

Reply:
@danwinship, thanks. I updated the content and preview here:

https://multicast-with-multitenant--ocpdocs.netlify.com/openshift-enterprise/latest/networking/openshift-sdn/using-multicast.html#nw-about-multicast_using-multicast

Reply:
@danwinship, I changed the wording and posted a new preview link above.

Reply:
sure